### ISSUE TITLE
Table bolt_x is not the correct schema: invalid column datepublish

### ISSUE THREAD

**Issue by rhyst** [bolt/bolt#7838]
Our web server recently upgraded to MariaDB 10.4.12 from 10.1.10 and this seems to be causing issues with the bolts database checking. I think similar issues have happened in the past. The error is of the form:

```
Table bolt_articles is not the correct schema: invalid column datepublish, invalid column datedepublish, invalid column ownerid, invalid column templatefields, invalid column title, invalid column date, invalid column template
```
And appears for every table. This is using Bolt 3.6.11. Clicking Update Database does nothing.

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | 3.6.11
| Install type                   |  Tar flat install
| BC Break                     | ?
| PHP version                 | 5.6.40
| Web server                  | Apache

Thanks

**Comment by bobdenotter** [comment]
In short: Your DB version is significantly newer than your PHP version. Best way to fix is to update PHP to 7.1+, and then run `composer update` so you'll get the appropriate versions of the Doctrine libs, and I'm confident it'll work properly then.

**Comment by rhyst** [comment]
Thanks for the quick response, I will look into getting it updated. Will Bolt just use php 7 is it is available or is there any config to do?
